rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -1122,7 +1130,7 @@ public boolean restartInErrorTask(@HeaderParam("sessionid") String sessionId, @P
             @PathParam("jobid") String jobId, @PathParam("prefix") String prefix)
             throws NotConnectedRestException, UnknownJobRestException, PermissionRestException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)